pulp_ansible/app/migrations/0046_add_fulltext_search_fix.py

```diff
@@ -85,12 +82,11 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
-        migrations.RunSQL(
-            sql=POPULATE_COLLECTIONS_TS_VECTOR,
-            reverse_sql=migrations.RunSQL.noop,
-        ),
         migrations.RunSQL(
             sql=CREATE_COLLECTIONS_TS_VECTOR_TRIGGER,
             reverse_sql=DROP_COLLECTIONS_TS_VECTOR_TRIGGER,
+        ),
+        migrations.RunSQL(
+            sql=REBUILD_COLLECTIONS_TS_VECTOR,
```

Comment:
Does there need to be a noop for the reverse_sql? I think you can't down-migrate if there is no reverse.

Reply:
Probably.